src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    getPeople()
+      .then(data => setPeople(data))
+      .catch(error => {
+        setErrorMessage('Something went wrong');
+        throw error;
```

Comment:
Throwing an error inside the `catch` block will propagate the error and may cause unhandled promise rejections. Instead of throwing the error, consider logging it or handling it in a way that doesn't disrupt the application flow.